easybuild/easyconfigs/o/OCaml/OCaml-4.01.0-ictce-5.5.0.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Luxembourg Centre for Systems Biomedicine
+# Authors::   Fotis Georgatos <fotis.georgatos@uni.lu>
+# License::   MIT/GPL
+# $Id$
+##
+
+name = "OCaml"
+version = "4.01.0"
+
+homepage = 'http://ocaml.org/'
+description = """OCaml is a general purpose industrial-strength programming language
+ with an emphasis on expressiveness and safety. Developed for more than 20 years at Inria
+ it benefits from one of the most advanced type systems and supports functional,
+ imperative and object-oriented styles of programming."""
+
+toolchain = {'name': 'ictce', 'version': '5.5.0'}
+
+sources = ['ocaml-%(version)s.tar.gz']
+sourcedir = name.lower() + '-' + '.'.join(version.split('.')[0:-1])
+source_urls = ['http://caml.inria.fr/pub/distrib/%s' % sourcedir]
+
+patches = ['OCaml-%(version)s_icc-fixes.patch']
+
+builddependencies = [('Autoconf', '2.69')]
+dependencies = [('ncurses', '5.9')]
```

Comment:
I come to realized that if ncurses is there, I bet `libreadline`  should also be. ditto for goolf. ldd will prove the case